tests/Aspire.Azure.Search.Documents.Tests/AspireAzureSearchExtensionsTests.cs

```diff
@@ -0,0 +1,71 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Azure.Search.Documents.Indexes;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Xunit;
+
+namespace Aspire.Azure.Search.Documents.Tests;
+
+public class AspireAzureSearchExtensionsTests
+{
+    private const string SearchEndpoint = "https://aspireazuresearchtests.search.windows.net/";
+    private const string ConnectionString = $"Endpoint={SearchEndpoint};Key=fake";
```

Comment:
nit: make these local to `ReadsFromConnectionStringsCorrectly` so there is no unintentional overlap with the other test.